index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title> my portfolio</title>
+</head>
+
+<body>
+    <main class="container">
+        <section>
+            <a class="logo">logos</a>
+            <nav class="humbuger"><img src="images/icones/Union.png" alt="humburger"> </nav>
+        </section>
+        <div class="contain">
+            <div class="container-title">
+                <h1 class="item1">Faustin Makutano</h1>
+                <h2 class="item2">Software Developer</h2>
+                <div class="literature">
+                    <p class="item3">I can help you build a product , feature or website Look through some of my work and experience!<br> If you like what you see and have a project you need coded,<br> don’t hestiate to contact me.</p>
+                </div>
+            </div>
+        </div>
+        <header class="border-bottom">
+            <ul class="skills-list">
+                <li class="language">
+                    <h3 class="skill-title">Language</h3>
+                    <ul class="sublist">
+                        <li class="skill">JavaScript</li>
+                        <li class="skill">Ruby</li>
+                        <li class="skill">Html</li>
+                        <li class="skill">Css</li>
+                    </ul>
+                </li>
+                <li class="frameworks">
```

Comment:
- [ ] It is good that you updated this section to use the more semantically appropriate tags of `<ul> and <li>` as indicated by the previous reviewer. However, you should remove the list style `dots` and margins at the left to make it look like the one in the design. (list-style: none;)
#### Design
![image](https://user-images.githubusercontent.com/44057863/170760659-629abe11-5a5c-45c1-85ee-5c0e94ec8c03.png)

#### Impl
![image](https://user-images.githubusercontent.com/44057863/170760609-1d3c5904-7824-4778-8af8-b4fe41e255d3.png)

- [ ] You should add the section separator line at the top of this section. You can use `<hr>` tag for that.